src/main/scala/beam/agentsim/agents/modalbehaviors/DrivesVehicle.scala

```diff
@@ -870,7 +876,14 @@ trait DrivesVehicle[T <: DrivingData] extends BeamAgent[T] with Stash {
       0.0
   }
 
+  /**
+    * process the event ChargingPlugInEvent
```

Comment:
As mentioned above, should the start/end still be here or be in CNM? Is that what this comment is about? When _would_ be the time to delete it if not during this PR?

Reply:
That will happen once we refractor Ridehail as well. We are waiting for a Colin's patch to merge first.